plugins/org.eclipse.emf.diffmerge.sirius/src/org/eclipse/emf/diffmerge/sirius/SiriusImageHelper.java

```diff
@@ -37,6 +40,12 @@ public class SiriusImageHelper {
   /** @see org.eclipse.emf.diffmerge.sirius.SiriusImageHelper getMainProjectName */
   private HashMap<EObject, String> _mainProjectNames = new HashMap<>();
 
+  private ComparisonRootContainerHelper comparisonRootContainerHelper;
+
+  public SiriusImageHelper(IComparison comparison) {
```

Comment:
maybe keep the original no args constructor or can you update as well below:
SiriusImageHelper is used in capella 
https://github.com/eclipse/capella/search?q=SiriusImageHelper